### PR TITLE
chore: adds ko support and automated release support

### PR DESCRIPTION
**Pull request by zhill**
Adds usage of ko for builds via Makefile targets as well as skaffold support.

### CODE REVIEW

**Comment by zhill** [comment]
Note that I would like to have better tests end-to-end but I'll look at that after this update since it requires thinking about the chart and how that is setup as well.

**Review by bradleyjones — APPROVED** [review]
Looks good to me

File: .ko.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+builds:
+  - id: anchore-admission-controller
+    main: ./cmd/kubernetes-admission-controller
+    env:
+      - CGO_ENABLED=0
+    flags:
+      - -tags
+      - -trimpath
```
Comment (imjasonh) [inline .ko.yaml:8]: AFAIK these are unneeded. `ko` already builds with `CGO_ENABLED=0` and `-trimpath` by default, and I think `-tags` without any values is a no-op.

**Review by imjasonh — APPROVED** [review]
Very nice!

File: .ko.yaml (hunk 1 above)
Comment (zhill) [reply .ko.yaml:8]: ack! Thanks!

**Pull request merged**